### ISSUE TITLE
Spotlight: Results view UI

### ISSUE THREAD

**Issue by mattkrick** [ParabolInc/parabol#5480]
Predecessors: https://github.com/ParabolInc/parabol/issues/5183.

Now that the results view is in place, let's make it look good

TEST CASES
- [ ] start with 10 reflection groups & open the spotlight of Card 1
- [ ] In other browser, drag a card that does not exist in the spotlight. whether that dragged card appears above or below the spotlight depends on the targetId of the drag. If the targetId refers to a card in the spotlight, it's above. else, below. there is an opacity transition if it changes
- [ ] When my spotlight is open, that source cards appears to be being dragged by me to everyone else
- [ ] put 2 cards in a column, each card has 6+ lines. click spotlight on the top card. close the spotlight modal. the cards keep their space. see the end of https://www.loom.com/share/62c17852fe784ae7b0e670deee3ee394

AC
- [ ] The reflection groups are laid out in a masonry grid with an 8px gutter (8px between 2 cards that are next to each other, 8px bottom padding)
- [ ] The masonry grid supports cards of varying heights (1-4 lines of text). There is no more than 8px between a reflection & the card beneath it
- [ ] a remote reflection is smooth & predictable (see above test cases)
- [ ] the spotlight is not a dropzone

Estimate: 8 hours

**Comment by nickoferrall** [comment]
Hey @mattkrick, just want to check whether you're thinking this issue will be in addition to the Groups UI issue that I created https://github.com/ParabolInc/parabol/issues/5390 and the PR https://github.com/ParabolInc/parabol/pull/5402? 

There are plenty of helpful additions you've included that don't exist in the Groups UI PR, e.g. the remote reflection drags are currently buggy, there's no opacity transition, and there's more than 8px of bottom padding.

A couple of questions:

1. Do you feel that the masonry grid is a requirement? It looks like the [designs in Figma](https://www.figma.com/file/u5bdWUn9hxkdvS2YOYDnXU/Spotlight?node-id=0%3A1) don't use a masonry grid.

2. > When my spotlight is open, that source cards appears to be being dragged by me to everyone else

    I think this test case is included in this issue https://github.com/ParabolInc/parabol/issues/5272 so perhaps I can remove it from this issue to keep PRs as small as possible?

I'd be happy to get started on this issue once I've completed your feedback in the Null Case PR. The current branch train for my Spotlight PRs is Null Case PR -> Groups UI -> Source to Groups -> Groups to Source. I plan on branching out of the Groups to Source branch so that it's using the latest code.

**Comment by mattkrick** [comment]
Oh I'm not sure where this goes, I just wanted to capture things as I went along to make sure we get the polish added before we ship to GA.

> Do you feel that the masonry grid is a requirement? It looks like the designs in Figma don't use a masonry grid.

Keen eye! Looks like figma uses vertical columns (top-to-bottom, left-to-right). This is _really_ difficult to implement with animations because if you moved the first card (the one in the top left) then every single card would have to shift & those cards that are at the top of a column would have to move to the bottom of the previous column. All that noise makes it feel kinda bad (this was originally how I built the grouping phase!). Happy to explore other options if there's an easier path

> I think this test case is included in this issue

Yep absolutely. let's just ignore this issue for now until we're ready to ship to GA & then we'll just use the stuff captured here as some integration tests
